.flake8/azul_flake8.py

```diff
@@ -305,6 +305,58 @@ def run(self):
         return visitor.errors
 
 
+class AzulLines:
+    name = 'azul_lines'
+    version = 1.0
+
+    begin_skip = '# noqa lines: begin'
+    end_skip = '# noqa lines: end'
+
+    def __init__(self, tree, lines):
+        # We need tree as a parameter otherwise run() won't be called
+        self.lines = lines
+
+    def run(self):
+        errors: List[ErrorInfo] = []
+        for num, line in self.unskipped_lines():
+            error = self.check_length(num, line)
+            if error is not None:
```

Comment:
Since `None` is more explicit, I think I prefer to keep this as is. It doesn't apply in this particular case, but sometimes, you can skip over falsey values like this by accident, for example if `error` here was a string and once was the empty string.